dec14_exercise.py

```diff
@@ -0,0 +1,60 @@
+ages = [
+29,
+19,
+17,
+26,
+32,
+16,
+21,
+
+]
+#4
+print(ages)
+#6
+print(ages[4])
+#7
+print(ages[-2])
+#8
+print(ages[2:6])
+#9
+print(ages[:3])
+#10
+if 17 in ages:
+    print(True)
+else:
+    print(False)
+#11
+if 42 in ages:
+    print(True)
+else:
+    print(False)
```

Comment:
Try solving each of these in a single statement. Take a look back at the `ds-prep-python-variables` exercise if you're not sure how:

---

![image](https://user-images.githubusercontent.com/16143771/146121304-f59e6163-d3f5-4a85-85e4-a7daedecb907.png)